release-notes/SupportedFeatures-v1.0.0.md

```diff
@@ -34,23 +34,22 @@ Supported features
 |              |FaultContract          |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
 |Security      |None                   |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
 |              |Transport              |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
-|              |Message                |[:no_entry_sign:](#known_issues)     |[:no_entry_sign:](#known_issues)    |[:no_entry_sign:](#known_issues)           | [:no_entry_sign:](#known_issues)    |
+|              |Message                |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)     |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)    |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)           | [:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)    |
 |Client Authentication|Basic           |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
-|              |Digest                 |:white_check_mark:                   |:white_check_mark:                  |[:no_entry_sign:](#known_issues)           | [:no_entry_sign:](#known_issues)    |
-|              |NTLM                   |:white_check_mark:                   |:white_check_mark:                  |[:no_entry_sign:](#known_issues)           | [:no_entry_sign:](#known_issues)    |
-|              |Kerberos/TCP           |[:no_entry_sign:](#known_issues)     |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
-|              |Kerberos/HTTP          |:white_check_mark:                   |:white_check_mark:                  |[:no_entry_sign:](#known_issues)           | [:no_entry_sign:](#known_issues)    |
+|              |Digest                 |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :grey_question:|
```

Comment:
Digest domain test passed on Linux when the server machine is domain joined. Linux machine can communicate with Redmond domain joined test server, OSX cannot.

It's unknown why domain joined server does not hit issue  #1332.